StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetAppearance.swift

```diff
@@ -306,5 +308,11 @@ public extension PaymentSheet {
                 public var spacing: CGFloat = 12.0
             }
         }
+        var analyticPayload: [String: Any] {
```

Comment:
Nit but wdyt about defining this and the `static let` above in an extension in `STPAnalyticsClient+PaymentSheet`?

Reply:
@porter-stripe Outside scope of this PR  and embedded too really but I wonder if it'd be more useful to send the actual value of these things.